lib/ansible/config/manager.py

```diff
@@ -34,131 +22,171 @@
 from ansible.utils.path import unfrackpath
 from ansible.utils.path import makedirs_safe
 
-
+Plugin = namedtuple('Plugin','name type')
+Setting = namedtuple('Setting','name value origin')
+
+IGNORE_CONFIGS = frozenset([None, '/dev/null'])
```

Comment:
no, had to add the string as that is what test/ uses all over, still need to research why, but for now its  here to make tests pass.